### ISSUE TITLE
ReplaySubjectConcurrencyTest > testReplaySubjectConcurrentSubscriptions FAILED

### ISSUE THREAD

**Issue by akarnokd** [ReactiveX/RxJava#4028]
```
rx.subjects.ReplaySubjectConcurrencyTest > testReplaySubjectConcurrentSubscriptions FAILED
    java.lang.RuntimeException: Concurrency Bug
        at rx.subjects.ReplaySubjectConcurrencyTest.testReplaySubjectConcurrentSubscriptions(ReplaySubjectConcurrencyTest.java:235)
```

**Comment by akarnokd** [comment]
New crash type:

```
rx.subjects.ReplaySubjectConcurrencyTest > testReplaySubjectConcurrentSubscriptions FAILED
    java.lang.NullPointerException
        at rx.subjects.ReplaySubjectConcurrencyTest.testReplaySubjectConcurrentSubscriptions(ReplaySubjectConcurrencyTest.java:221)
```

**Comment by akarnokd** [comment]
Fixed via #4051.
